lib/catalog/martyrology.ts

```diff
@@ -3532,6 +3829,17 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Philip of Jesus de las Casas',
       titles: [Title.Martyr],
     },
+    // src:
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/bienheureux-pierre-adrien-toulorge-martyr
+    // - https://en.wikipedia.org/wiki/Pierre-Adrien_Toulorge
+    pierre_adrien_toulorge_priest: {
```

Comment:
Norbertines who celebrate this celebration too, translated his first names into English (cf [this](https://www.norbertines.org.uk/blog/let-your-yes-be-yes-and-your-no-be-no) and [this](https://norbertines.org/saints-blesseds/)). They use both `Peter-Adrian Toulorge` and `Peter Adrian Toulorge`; I prefer the latter.

```suggestion
    peter_adrian_toulorge_priest: {
```

Reply:
I still wonder to what extent we should anglicize all the names, especially when it's a local saint. In fact, in this repository, many names aren't anglicized (we keep the original name).

Reply:
> I still wonder to what extent we should anglicize all the names

That is a good question. I don’t know the answer, but I try to follow the way it is written on the Internet. Even in the celebrations of persons with French names inscribed in a calendar of an English-speaking country, some saints are anglicised, others not.

I don’t want to forcefully push anglicisation of all French names, however, in this case, the source is a website of an order which also celebrates this particular blessed (in every country, therefore in English-speaking countries too), therefore, I believe we should ‘follow the lead’ in this case.

IMHO we should translate names if we find some relevant sources that it is done so, otherwise we should stick to its original spelling.

> especially when it's a local saint

While this blessed is indeed a local one, however, he is celebrated worldwide by Norbertines/Premonstratensians (even [in Slovakia](https://lh.kbs.sk/cgi-bin/l.cgi?qt=pdt&d=13&m=10&r=2023&k=skopraem) where the celebration name is `Bl. Petra Adriána Toulorge, kňaza a mučeníka z našej rehole` or `Blessed Peter Adrian Toulorge, Priest and Martyr from Our Order`; note that his name is slovakised too).